guide/sections/part1/data-consumer.adoc

```diff
@@ -29,19 +27,19 @@ Notifications from WIS2 Nodes and Global Caches are published on different topic
+Core data will be available from both a WIS2 Node and Global Caches, each of which publish a different notification message advertising alternative locations from where the data can be downloaded. Because these are _different_ messages, they will have _different_ identifiers. However, the each of these messages refers to the same data object, which is uniquely identified in the notification message using the ``data_id`` property. Notification messages from different sources can easily be compared to determine if they refer to the same data. By subscribing to the ``cache`` root topic, Data Consumers will only receive notifications about data available from the Global Caches. The ``origin`` root topic should be used when subscribing to notifications about Recommended data. Data Consumers should not subscribe to the ``origin`` root topic for notifications about Core data because notification messages provided on these topics will refer to data published on directly on the WIS2 Nodes (i.e., the "origin").
 
-Data Consumers need to consider their strategy for managing these duplicate messages. From a data perspective, it does not matter which Global Cache instance is used – they will all provide an identical copy of the data object published by the originating WIS2 Node. The simplest strategy is to accept the first notification message and download from the Global Cache instance that the message refers to (i.e., using a URL for the data object at that Global Cache instance). Alternatively, a Data Consumer may have a preferred Global Cache instance, for example, that is located in their region. Whichever Global Cache instance is chosen, Data Consumers will need to implement logic to discard duplicate notification messages. Also, note that Data Consumers should download from a Global Cache in preference to the originating WIS2 Node because Global Caches are designed to handle large loads.
+Data Consumers need to consider their strategy for managing these duplicate messages. From a data perspective, it does not matter which Global Cache instance is used – they will all provide an identical copy of the data object published by the originating WIS2 Node. The simplest strategy is to accept the first notification message and download from the Global Cache instance that the message refers to (i.e., using a URL for the data object at that Global Cache instance). Alternatively, a Data Consumer may have a preferred Global Cache instance, for example, that is located in their region. Whichever Global Cache instance is chosen, Data Consumers will need to implement logic to discard duplicate notification messages based on ``id`` and ``data_id``.
```

Comment:
If I'm being pedantic...

to discard duplicate notification messages based on ``id`` and duplicate data objects based on ``data_id``.

Reply:
Text updated as per suggestion.